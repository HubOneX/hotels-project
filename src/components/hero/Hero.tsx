import styled from "@emotion/styled";
import { Typography } from "@mui/material";
import React from "react";
import colors from "../../constants/colors";
import { FlexColumnBox } from "../utilityComponents/styledComponents";

const HeaderBox = styled(FlexColumnBox)`
  background: linear-gradient(159deg, ${colors.blue} 0%, ${colors.pink} 130%);
  width: 100%;
  height: 350px;
  box-shadow: 0px -1px 30px 0px ${colors.midGrey};
`;

const Hero = () => {
  // as discussed in the email conversation this part doesn't have an image from the API call
  return (
    <HeaderBox>
      <Typography variant="h1">HotelsFindr</Typography>
      <Typography variant="h2">
        This is a place for a great catchphrase
      </Typography>
    </HeaderBox>
  );
};

export default Hero;
