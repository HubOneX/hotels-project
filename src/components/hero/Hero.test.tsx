import React from "react";
import renderer from "react-test-renderer";
import Hero from "./Hero";

describe("Hero", () => {
  it("should match the snapshot", () => {
    const container = renderer.create(<Hero />);

    expect(container).toMatchSnapshot();
  });
});
