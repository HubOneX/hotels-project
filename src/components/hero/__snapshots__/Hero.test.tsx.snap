// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero should match the snapshot 1`] = `
<div
  className="MuiBox-root css-ehgow7"
>
  <h1
    className="MuiTypography-root MuiTypography-h1 css-o2w69a-MuiTypography-root"
  >
    HotelsFindr
  </h1>
  <h2
    className="MuiTypography-root MuiTypography-h2 css-1sra7t5-MuiTypography-root"
  >
    This is a place for a great catchphrase
  </h2>
</div>
`;
